recipe/meta.yaml

```
@@ -13,34 +13,41 @@ build:
   entry_points:
     - ydata_profiling = ydata_profiling.controller.console:main
     - pandas_profiling = ydata_profiling.controller.console:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
   number: 0
+  skip: True   #  [py<37]
 
 requirements:
   host:
-    - python >=3.7,<3.11
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7,<3.11
-    - scipy >=1.4.1,<1.10
+    - python
+    - scipy=1.9.3
```

```suggestion
    - scipy >=1.4.1,<1.10
```
See https://github.com/ydataai/ydata-profiling/blob/v4.1.1/requirements.txt#L1

---

Thanks Charles!